.gitignore

```diff
@@ -35,13 +35,11 @@ xcuserdata
 # Packages/
 .build/
 
-# CocoaPods
-#
-# We recommend against adding the Pods directory to your .gitignore. However
-# you should judge for yourself, the pros and cons are mentioned at:
-# https://guides.cocoapods.org/using/using-cocoapods.html#should-i-check-the-pods-directory-into-source-control
-#
-# Pods/
+# While ReSwift has no dependencies we are still using CocoaPods, but only
+# to ensure developers use the same binary version of SwiftLint. Since
+# this is just a development dependency it is ok to ignore it, and rely on
+# developers to generate it when required.
+Pods/
```

Comment:
I think it makes sense to ignore the Pods folder for the sake of keeping the repo lean.